vault/seal/awskms/awskms_test.go

```diff
@@ -59,6 +59,38 @@ func TestAWSKMSSeal_Lifecycle(t *testing.T) {
 	}
 }
 
+// This test executes real calls. The calls themselves should be free,
```

Comment:
What's the difference between this and the lifecycle test right above it (`TestAWSKMSSeal_Lifecycle`)?

Reply:
> This test executes real calls